libs/libarchfpga/src/graph_type.h

```diff
@@ -0,0 +1,13 @@
+#ifndef GRAPH_TYPE_H
```

Comment:
May consider to rename this file to ``rr_graph_type.h``.
Just a note here. We will finalize it only after a discussion with Vaughn